int32 ac_id
time time
geometry_msgs/Point position
---
Wind wind
